### PR TITLE
[DOCFIX] Fix terms in CN docs

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

**Comment by apc999** [comment]
alluxio-bot, check this please

File: docs/cn/Contributing-Getting-Started.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: global
-title: 开始贡献源码
-nickname: 开始贡献源码
+title: 开发者上手指南
+nickname: 开始者上手指南
```
Comment (ifcharming) [inline docs/cn/Contributing-Getting-Started.md]: nit : s/开始者/开发者

**Comment by ifcharming** [comment]
LGTM, with one comment on typo

**Comment by apc999** [comment]
thanks. fixed

File: docs/cn/Contributing-Getting-Started.md (hunk 1 above)
Comment (apc999) [inline docs/cn/Contributing-Getting-Started.md]: done

**Comment by ifcharming** [comment]
LGTM

**Pull request merged**